tools/metric_itlb_mpki

```diff
@@ -42,17 +46,21 @@ function init_itlb_mpki() {
 
 function calc_itlb_default_mpki() {
   local perf_data_file="$1"
+  local a=`return_pmu_value "itlb_misses.walk_completed" ${perf_data_file}`
+  local b=`return_pmu_value "instructions" ${perf_data_file}`
+
+  local verbose_a=`expand_if_verbose "[$a]"`
```

Comment:
Ah, having now discovered the verbose option and what it already produced, such as:

```
 Performance counter stats for './run.sh':

    27,506,199,587      cycles:u
    19,794,238,994      instructions:u            #    0.72  insn per cycle
         2,564,841      itlb_misses.walk_completed:u
         2,564,841      itlb_misses.walk_completed_4k:u
                 0      itlb_misses.walk_completed_2m_4m:u
                 0      itlb_misses.walk_completed_1g:u

      60.006276325 seconds time elapsed

       9.375289000 seconds user
      24.967090000 seconds sys
```

these additions (of the counters post-pending the formulas) may be un-necessary.
What we do need is *documentation* though... to make it clear what features are available and how they can help diagnose runs.

Reply:
@grahamwhaley How this is different than existing -v option?

Reply:
@grahamwhaley  I like the idea of help function in each metric script which will be called from main measure-perf script. Should we expand to all the metric scripts?

Reply:
> @grahamwhaley How this is different than existing -v option?

it's not - which I only found/realised later. As noted, I think we can drop the verbosity commit here. We need better docs.....

Reply:
> @grahamwhaley I like the idea of help function in each metric script which will be called from main measure-perf script. Should we expand to all the metric scripts?

Yes, I would update for all the scripts, as noted in the PR message. I made this a simple one off and RFC to get feedback before investing the effort.

I'll drop the verbose commit and add the descriptions for the rest of the scripts and then PR an update for review. thanks!